### ISSUE TITLE
Search window size is not remembered

### ISSUE THREAD

**Issue by kumar007git** [adirh3/Fluent-Search#678]
**Describe the bug**
I resize the search window size but when fluent search is started the next time then search window size is not remembered (it's not the same as last time)

**To Reproduce**
1. Call up search window
2. Increase the search window size
3. Exit fluent search
4. Start fluent search
5. call up search window

**Expected behavior**
Search window size should be same as before.

**Desktop (please complete the following information):**
 - Windows 10 Version: 21H2 (64-bit)
 - Fluent Search Version 0.9.92.44

**Comment by adirh3** [comment]
@kumar007git did you enable `Remember window size after restart`?

**Comment by kumar007git** [comment]
Yes, it is enabled. I tried it many times again before typing this reply and it still does the same thing.
I resized the search window to make it large.
I exited fluent search.
I started fluent search.
The search window size is not the same as the large one. It is still small.

**Comment by kumar007git** [comment]
after testing so many times, I am 100% positive, remember window size is not working.

**Comment by adirh3** [comment]
@kumar007git can you please send me your settings in Discord?

**Comment by kumar007git** [comment]
Hello Adhir,
I don't use discord sorry. The 'remember window size after restart' appears to only apply to all windows of fluent search except the search window.

I adjust the search window size and fluent search remembers it but if I restart fluent search then the search window size is reset. 

I think this may be due to the search window style setting (which can be set to full window or search bar only) and when fluent search is started it is enforcing this setting and overriding/resetting the custom size I gave to the search window when I resized the search window. I believe the search window style setting is interfering with my resizing of the search window.
